src/main/java/org/folio/innreach/domain/service/impl/HoldingsServiceImpl.java

```diff
@@ -32,4 +40,10 @@ public Holding update(Holding holding) {
     return holding;
   }
 
-}
\ No newline at end of file
+  @Override
+  @Cacheable(cacheNames = HOLDING_SOURCE_CACHE)
+  public Optional<HoldingSourceDTO> findHoldingSourceByName(String sourceName) {
```

Comment:
you have to include "tenant" as part of cache id
see UserServiceImpl for examples